Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.10.5-slim-bullseye
+
+WORKDIR /usr/src/app
+
+RUN pip install --no-cache-dir scrap_engine
+
+COPY . .
```

For user comfort, I would bind the internal save volume to `/data`

```suggestion
RUN pip install --no-cache-dir scrap_engine && \
    mkdir -p /root/.cache/pokete && \
    ln -s /root/.cache/pokete /data

COPY . .

VOLUME ["/data"]
```